### DISCUSSION TITLE
How to add new menu and categories

### DISCUSSION THREAD

**Discussion by maheshdharhari** [unfolding-io/nebulix#37] [General]
I want to add new menu and redirect to it and new categories as well. Please help me someone.

**Comment by preetamslot** [comment]
Hi, sorry for the late reply.
Nebulix is a bit out of date and did not have the time for an update.

If all is well you can add menus with StaticCMS, just go to menus and add a new menu.
The output of this will be mdx:
https://github.com/unfolding-io/nebulix/blob/main/src/content/menu/drinks.mdx

I also have an more up to date and user friendly restaurant theme:
https://astrorante.unfolding.io/

This has a visual editor (Storyblok CMS) and has a free plan for 1 user.

**Reply by maheshdharhari** [reply]
I tried saving it, but even though I received the message 'Entry saved,' it didn't actually save. I have already published it on Netlify.

![image](https://github.com/user-attachments/assets/4171e0de-605f-4d24-bfea-8789f3fbc880)

I might be making some mistakes since I'm new to Astro. Could you please help me? @preetamslot 

![image](https://github.com/user-attachments/assets/7f563fdd-ad35-4ec5-88d8-78f5ef9caad3)
